### ISSUE TITLE
Re-appearance of symbols for latest dashboard change when a non-dashboard-affecting slider is moved

### ISSUE THREAD

**Issue by RobTerwel** [quintel/etmodel#1912]
Hard to phrase, but here is an attempt: 

Changing certain sliders does not affect the dashboard quantities at all; however, when doing so the increase/decrease arrows for the **previous** change re-appear in the dashboard.

This is best illustrated by an example. Set all trucks to be electric in the demand of car technology, and then change the slider for efficiency improvement `combustion engine trucks`. One would expect the dashboard quantities not to change at all, because there are no combustion engine trucks in existence. This is indeed observed; however, one does see the arrows reflecting the previous change (in this case, of setting all trucks to be electric) re-appear in the dashboard. This might give the user the impression that the quantities in the dashboard do change because of these symbols, while in fact they don't.

It's also worth noting that these symbols do *not* appear if enough time has passed between the previous change and changing a slider that does not affect the dashboard quantities.

Re-assign at will.

P.S. My sincerest apologies for this horrifying title.

**Comment by ChaelKruip** [comment]
This behaviour is as old as the ETM. I kinda got used to it but that is not a reason to correct it. It would be better IMO if the arrows disappear as you touch a new slider that does not change anything.

Assigning @antw to pick this up after the deploy.

**Comment by jorisberkhout** [comment]
Assigning @grdw .

**Comment by grdw** [comment]
Took my some reading but I get the issue. :sweat_smile: 

The current implementation is that it 'animates' away after [30s](https://github.com/quintel/etmodel/blob/master/app/assets/javascripts/lib/views/constraint_view.coffee#L124)

**Comment by grdw** [comment]
> IMO if the arrows disappear as you touch a new slider that does not change anything.

I would love that aswell but the event is triggered >5 times when dragging one slider. Also checking if a slider 'did nothing' overall is harder than I thought. I wasted too much time on this (consider it a "I probably need a very speedy introduction to Backbone in combination with Coffeescript").

The fix I came up with is not letting the arrows stay on the field for 30 seconds. That's too long. How about 2 seconds? That way you see that it changed and that's it.
